lib/vault/encrypted_model.rb

```diff
@@ -144,6 +131,19 @@ def _vault_validate_options!(options)
               "`:context' must take 1 argument!"
           end
         end
+        if transform_opts = options[:transform_secret]
+          if !transform_opts[:transformation]
+            if !transform_opts[:type] && !transform_opts[:template]
+              raise Vault::Rails::ValidationFailedError, "Option " \
+                "`:transform_secret' must be supplied with either a " \
+                "`:transformation' option or a `:type' and `:template!"
```

Comment:
If you provide `:type` and `:template`, does it create a transformation on the fly to use?

Reply:
I forgot to remove that comment. I decided not to support it as of right now.